README.md

```diff
@@ -102,11 +102,13 @@ Each time you need to consume recent EDS changes elsewhere, you need to bump the
 - Prompt you for the new version number
 - Update `package.json` with the version number
 - Commit the change in a new commit
+- Get your PR approved in your branch and merge it to master
 
-2. To create and push a git tag, do one of these:
+2. To create and push a git tag:
 
-- Run `git tag v1.2.3 && git push origin v1.2.3` on your branch.
-- Create a release `v1.2.3` on GitHub, if it's already in trunk.
+- Switch to `master` branch and pull in your latest change
+- Run `git push origin v{new version number}` this will create the new tag with the version number in Github
+- Draft a new release in Github using the new tag
```

Comment:
What's the value in doing these releases? They aren't technically required, only the package.json and git tag combined are needed by consumers of EDS.

Reply:
I know we already had it in the readme, not sure why we started doing it is all.

Reply:
I suppose creating a new release with the most recent changes since the previous release is more for bookkeeping?

Reply:
Yeah fair enough. I guess the tags just seem to do that job to some extent as well already.